app_dart/lib/src/service/config.dart

```diff
@@ -46,6 +46,7 @@ class Config {
     flutterSlug,
     packagesSlug,
     pluginsSlug,
+    impellerSlug,
```

Comment:
Do we also want to support the repo in the build dashboard, like saving commits/tasks to datastore?